package.json

```
@@ -5,9 +5,14 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint --ext .js .",
-    "lint:fix": "eslint --ext .js . --fix"
+    "lint:fix": "eslint --ext .js . --fix",
+    "playground:dev": "yarn --cwd ./playground && yarn --cwd ./playground develop",
+    "playground:build": "yarn --cwd ./playground build",
+    "playground:serve": "yarn --cwd ./playground serve"
```

```suggestion
    "playground:dev": "yarn --cwd ./playground && yarn --cwd ./playground develop",
    "playground:build": "yarn --cwd ./playground && yarn --cwd ./playground build",
    "playground:serve": "yarn --cwd ./playground && yarn --cwd ./playground serve"
```

the first part `yarn --cwd ./playground` is used to install the packages in case the user forgot to `yarn install` in the repository. Because it can happen in all three cases it is best to keep them everywhere